### PR TITLE
bullet points indented correctly

### PR DESCRIPTION
**Pull request by gumillie222**
#637

### CODE REVIEW

File: client/src/components/Markdown.jsx [hunk 1]
```diff
@@ -1,14 +1,35 @@
...
+  return <li className="mt-2 font-normal text-base text-gray-600">{children}</li>;
+}
 
 // the components prop takes an object mapping tag names to React components
 export function Markdown({ text }) {
   return (
     <div className="max-w-xl" id="markdown">
       <ReactMarkdown
-        components={{ h1: H1, h2: H2, h3: H3, h4: H4, ul: UL, li: LI, p: P, a: A}}
+        // components={{ h1: H1, h2: H2, h3: H3, h4: H4, ul: UL, li: LI, p: P, a: A}}
```
[the remark this answers is not in the document]
Comment (JamesPHoughton) [reply client/src/components/Markdown.jsx:31]: We may be able to deal with this by getting global H1 H2 etc styling set up using Windi

**Review by JamesPHoughton — COMMENTED** [review]
hey, lets talk a bit about this today. =)

**Comment by gumillie222** [comment]
fixed in branch https://github.com/Watts-Lab/deliberation-empirica/tree/james-millie-styling

**Comment by JamesPHoughton** [comment]
Closing in favor of https://github.com/Watts-Lab/deliberation-empirica/pull/773

**Pull request closed without merging**